UnityProject/Assets/Scripts/Objects/Machines/FermentingBarrel.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using Mirror;
+using AddressableReferences;
+using Effects;
+using Items;
+using Items.Food;
+using Systems.Botany;
+using Chemistry;
+using Chemistry.Components;
+
+namespace Objects.Kitchen
+{
+	/// <summary>
+	/// A barrel into which players can insert certain food items.
+	/// Upon being inserted, they will be fermented into certain reagents.
+	/// </summary>
+	public class FermentingBarrel : NetworkBehaviour
```

Comment:
```suggestion
	public class FermentingBarrel : MonoBehaviour
```

Cant see why its needs to be a networkbehaviour